src/__tests__/screenForTest.py

```diff
@@ -108,7 +101,16 @@ def getRowsForPastScreen(self, pastScreen):
         return self.getRows(screen=self.pastScreens[pastScreen])
 
     def getRowsWithAttributesForPastScreen(self, pastScreen):
-        return self.getRowsWithAttributes(screen=self.pastScreens[pastScreen])
+        if pastScreen is None:
+            pastScreen = [self.getNumPastScreens()]
+        elif not isinstance(pastScreen, list):
+            pastScreen = [pastScreen]
+
+        pages = map(lambda screenIndex: self.getRowsWithAttributes(
```

Comment:
woof, whats with all this introduced complexity? why is pastScreen now a list? :*(